### ISSUE TITLE
Declarations without code show up with 0% coverage

### ISSUE THREAD

**Issue by dcsobral** [mtkopone/scct#34]
Classes, traits and objects without concrete methods show up with 0% code coverage. I don't think that's mathematically correct (!code.exists(!_.covered) == code.forall(_.covered)), but, much worse than that, it distracts while looking over what's covered.

**Comment by mtkopone** [comment]
Yeah, probably something like "N/A" would be more appropriate for empty classes/traits/objects. Will look into it.
